packages/ibm-products-web-components/package.json

```
@@ -0,0 +1,81 @@
+{
+  "name": "@carbon/ibm-products-web-components",
+  "description": "Carbon for IBM Products Web Components",
+  "version": "0.0.0",
+  "license": "Apache-2.0",
+  "main": "es/index.js",
+  "module": "es/index.js",
+  "type": "module",
+  "bugs": "https://github.com/carbon-design-system/ibm-products/issues",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/carbon-design-system/ibm-products.git",
+    "directory": "packages/ibm-products-web-components"
+  },
+  "publishConfig": {
+    "access": "public",
+    "provenance": true
+  },
+  "keywords": [
+    "carbon",
+    "carbon design system",
+    "carbon community",
+    "carbon for ibm products",
+    "carbon for ibm products web components",
+    "web components"
+  ],
+  "scripts": {
+    "dev": "vite",
+    "build": "yarn clean && node tasks/build.js",
+    "build:storybook": "storybook build",
+    "clean": "rimraf es lib scss dist storybook-static",
+    "preview": "vite preview",
+    "storybook": "storybook dev -p 3000",
+    "test": "vitest run",
+    "test:ui": "vitest --ui"
+  },
+  "dependencies": {
+    "@carbon/ibm-products-styles": "^2.30.1",
+    "@carbon/styles": "1.63.1",
+    "@carbon/web-components": "2.13.0",
+    "lit": "^3.1.0"
+  },
+  "devDependencies": {
+    "@carbon/icons": "^11.50.0",
+    "@carbon/motion": "^11.23.0",
+    "@mordech/vite-lit-loader": "^0.35.0",
+    "@rollup/plugin-alias": "^5.1.1",
+    "@rollup/plugin-commonjs": "^28.0.0",
+    "@rollup/plugin-node-resolve": "^15.3.0",
+    "@rollup/plugin-typescript": "^12.1.0",
+    "@storybook/addon-essentials": "^8.2.8",
+    "@storybook/addon-links": "^8.2.8",
+    "@storybook/addon-storysource": "^8.2.8",
+    "@storybook/blocks": "^8.2.8",
+    "@storybook/manager-api": "^8.3.4",
+    "@storybook/theming": "^8.3.4",
+    "@storybook/web-components": "^8.2.8",
+    "@storybook/web-components-vite": "^8.2.8",
+    "@types/jest": "^29.5.13",
+    "@vitest/browser": "latest",
+    "@vitest/ui": "latest",
+    "autoprefixer": "^10.4.20",
+    "cssnano": "^7.0.6",
+    "eslint": "^9.11.1",
+    "eslint-config-carbon": "^3.17.0",
+    "globby": "^14.0.2",
+    "happy-dom": "^15.7.4",
+    "jsdom": "latest",
+    "postcss": "^8.4.47",
+    "remark-gfm": "^4.0.0",
+    "rimraf": "^6.0.1",
+    "rollup": "^4.23.0",
+    "rollup-plugin-copy": "^3.5.0",
+    "sass": "^1.51.0",
+    "storybook": "^8.2.8",
+    "storybook-addon-accessibility-checker": "^3.1.61-rc.3",
+    "typescript": "^5.5.3",
+    "vite": "^5.4.1",
+    "vitest": "latest"
```

```suggestion
    "vitest": "latest",
    "web-component-analyzer": "2.0.0"
```

Add `web-component-analyzer` package for build script change above.